cmd/api/routes.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"github.com/go-chi/chi"
+	"github.com/go-chi/chi/v5/middleware"
+	"github.com/go-chi/cors"
+	"github.com/vladyslavpavlenko/genesis-api-project/internal/handlers"
+	"net/http"
+)
+
+func routes() http.Handler {
+	mux := chi.NewRouter()
+	mux.Use(middleware.Logger)
+
+	mux.Use(cors.Handler(cors.Options{
+		AllowedOrigins:   []string{"https://*", "http://*"},
+		AllowedMethods:   []string{"GET", "POST", "PUT", "DELETE", "OPTIONS"},
+		AllowedHeaders:   []string{"Accept", "Authorization", "Content-Type", "X-CSRF-Token"},
+		ExposedHeaders:   []string{"Link"},
+		AllowCredentials: true,
+		MaxAge:           300,
+	}))
+
+	mux.Use(middleware.Heartbeat("/"))
+
+	mux.Get("/rate", handlers.Repo.GetRate)
```

Comment:
how abt versioning ur api? like v1/ v2/

Reply:
Completely agree with you on this. Just changed it to `/api/v1/...`. I wanted to stick precisely to the requirements of the test task, so I decided not to version my API at first. Thanks for reminding me!